gpii/node_modules/settingsHandlers/test/WebSocketsSettingsHandlerTests.js

```diff
@@ -0,0 +1,125 @@
+/*
```

Comment:
I added tests for all this, you can find them [here](https://github.com/javihernandez/universal/blob/GPII-989/gpii/node_modules/flowManager/test/BrowserChannelTests.js).